rspace-bench/src/test/scala/coop/rchain/rspace/bench/ExampleDataBasicBench.scala

```diff
@@ -0,0 +1,77 @@
+package coop.rchain.rspace.bench
+
+import java.nio.file.{Files, Path}
+
+import cats.Id
+import cats.effect._
+import coop.rchain.rspace.examples.StringExamples._
+import coop.rchain.rspace.examples.StringExamples.implicits._
+import coop.rchain.rspace.history.Branch
+import coop.rchain.rspace.util._
+import coop.rchain.rspace.{LMDBStore, _}
+import org.openjdk.jmh.annotations.{Benchmark, Scope, State, TearDown}
+import coop.rchain.shared.PathOps.RichPath
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ExampleDataBasicBench {
+
+  import ExampleDataBasicBench._
+
+  @Benchmark
+  def consumeProduce(state: BenchState): Unit = {
```

Comment:
This benchmark is useless because of the default java serialization. Let's not keep it

Reply:
<o